src/main/java/org/ow2/proactive/catalog/report/TableDataBuilder.java

```diff
@@ -0,0 +1,178 @@
+            }
+
+            Row<PDPage> dataRow = table.createRow(10f);
+            createDataCell(dataRow, (100 / 8f), catalogObject.getBucketName());
+            createDataCell(dataRow, (100 / 8f), getProjectName(catalogObject));
+            createDataCell(dataRow, (100 / 8f), catalogObject.getName());
+            createDataCell(dataRow, (100 / 8f) * 2, getDescription(catalogObject));
+            createDataCell(dataRow, (100 / 8f), catalogObject.getKind());
+            createDataCell(dataRow, (100 / 8f), catalogObject.getContentType());
+            createIconCell(dataRow, getIcon(catalogObject));
+
+        }
+        if (orderedObjectsPerBucket.isEmpty()) {
+            createEmptyTableRow(table);
+        }
+
+    }
+
+    private void createDataHeader(BaseTable table) {
+        Row<PDPage> factHeaderrow = table.createRow(15f);
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Bucket Name");
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Project Name");
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Object Name");
+        createDataHeaderCell(factHeaderrow, (100 / 8f) * 2, "Description");
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Kind");
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Content Type");
+        createDataHeaderCell(factHeaderrow, (100 / 8f), "Icon");
+        table.addHeaderRow(factHeaderrow);
+    }
+
+    private void createDataHeaderCell(Row<PDPage> factHeaderrow, float width, String title) {
+        Cell<PDPage> cell;
+        cell = factHeaderrow.createCell(width, title);
+        cell.setFontSize(8);
+        cell.setFillColor(java.awt.Color.decode(LIGHT_CYAN));
+        cell.setAlign(HorizontalAlignment.CENTER);
+    }
+
+    private void createDataCell(Row<PDPage> row, float width, String catalogObjectData, int size) {
+        createDataCell(row, width, catalogObjectData, size, WHITE);
+    }
+
+    private void createDataCell(Row<PDPage> row, float width, String catalogObjectData, int size, String color) {
+        Cell<PDPage> bucketNameCell = row.createCell(width, catalogObjectData);
+        bucketNameCell.setFontSize(size);
+        bucketNameCell.setAlign(HorizontalAlignment.CENTER);
+        bucketNameCell.setFillColor(java.awt.Color.decode(color));
+    }
+
+    private void createDataCell(Row<PDPage> row, float width, String catalogObjectData) {
+        createDataCell(row, width, catalogObjectData, 6);
+    }
+
+    private String getProjectName(CatalogObjectMetadata catalogObject) {
+        return Optional.ofNullable(catalogObject.getProjectName()).orElse("");
+    }
+
+    private String getDescription(CatalogObjectMetadata catalogObject) {
+        return catalogObject.getMetadataList()
+                            .stream()
+                            .filter(metadata -> metadata.getKey().equals("description"))
+                            .map(metadata -> metadata.getValue())
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)